source/SIL.AppBuilder.Portal/common/prisma/schema.prisma

```diff
@@ -484,6 +484,8 @@ model WorkflowDefinitions {
   RepublishWorkflows   ProductDefinitions[] @relation("ProductDefinitions_RepublishWorkflowIdToWorkflowDefinitions")
   Workflows            ProductDefinitions[] @relation("ProductDefinitions_WorkflowIdToWorkflowDefinitions")
   StoreType            StoreTypes?          @relation(fields: [StoreTypeId], references: [Id], onDelete: Restrict, onUpdate: NoAction, map: "FK_WorkflowDefinitions_StoreTypes_StoreTypeId")
+  ProductType          Int                  @default(0)
+  AdminRequirements    Int[]                @default([0])
```

Comment:
Do you want the default to be an array containing a value of 0 or an empty array? Is it possible to have an empty array?

Reply:
The reason I chose to do an array with a zero rather than an empty array is because zero is the value of one of the enum members. However, given that there should never be a workflow definition where an empty array could be meaningfully distinct from an array with the zero in it, I suppose that that change could be made without too much issue?

Reply:
What does `[0]` indicate?

Reply:
It represents the absence of any specified WorkflowAdminRequirements. This is more important for the state/transition filtering and probably doesn't need to be specified in the WorkflowDefinitions table. So we could change to having an empty array as default and it would work the same if I treat an empty array the same as I would [WorkflowAdminRequirements.None (0)] in the code.

Reply:
OK. Then it is probably fine.